src/server.ts

```diff
@@ -0,0 +1,24 @@
+// Import the 'express' module along with 'Request' and 'Response' types from express
+import express, { Request, Response } from "express";
+import cors from "cors";
+// Create an Express application
+const app = express();
+
+// Specify the port number for the server
+//const PORT = process.env.PORT || 3002;
+const PORT = __filename.endsWith(".ts") ? 3003 : 3001;
```

Comment:
what is it for? do we document it on readme?

Reply:
this line is not available anymore, it was replaced with 

export const PORT = process.env.PORT || 3001 in config.ts file